plugins/scalafix/Scalafix.scala

```
@@ -46,19 +50,28 @@ object Scalafix{
           allowEmpty || rewrites.nonEmpty || patches.nonEmpty,
           "You need to provide some rewrites via: `override def scalafix = super.scalafix.copy( rewrites = Seq(...) )`"
         )
-        files.foreach{ case (from, to) =>
-          implicit val ( ctx, ctx2 ) = context(from)
-          lib.writeIfChanged(
-            to,
-            Patch(
-              (
-                patches
-                ++ rewrites.flatMap(
-                  _.rewrite( ctx2 ).to
-                )
-              ).to
+        files.foreach {
+          case (from, to) =>
+            implicit val (ctx, ctx2) = context(from)
+            println("printing sources")
+            println(ctx.mirror.sources)
+            println("printing database")
+            println(ctx.mirror.database)
+            println("printing mirror database messages")
+            println(ctx.mirror.database)
+            println("end printing")
+            assert(ctx.mirror.database.toString.trim != "", "I do not believe the database should be empty here")
```

I do believe fail fast would be better here, but this should be enough to advice to get me back on track. 👍